Lib/importlib/util.py

```diff
@@ -130,10 +142,14 @@ class allowing_all_extensions:
 
     In both cases, this context manager may be used to temporarily
     disable the check for compatible extension modules.
+
+    You can get the same effect as this function by implementing the
+    basic interface of multi-phase init (PEP 489) and lying about
+    support for mulitple interpreters (or per-interpreter GIL).
     """
 
-    def __init__(self, disable_check=True):
-        self.disable_check = disable_check
+    def __init__(self, disable_check):
```

Comment:
Without the keyword, `_incompatible_extension_module_restrictions(True)` reads to me like incompatible module restrictions are turned on. I would either make the parameter keyword-only or invert the meaning (i.e., `enable_check`).

Reply:
Good point.

Reply:
done